src/classes/CW_CampaignListSettingHelper.cls

```diff
@@ -0,0 +1,134 @@
+			campSetting.Name=sName;
+			campSetting.DateTime_Value__c=defaultVal;
+			insert campSetting;
+		}
+		return campSetting.DateTime_Value__c;
+	}
+	
+	public static Date getDateValue(String sName , Date defaultVal){
+		CW_CampaignListSetting__c campSetting=CW_CampaignListSetting__c.getValues(sName);
+		if(campSetting==null){
+			campSetting= new CW_CampaignListSetting__c();
+			campSetting.Name=sName;
+			campSetting.Date_Value__c=defaultVal;
+			insert campSetting;
+		}
+		return campSetting.Date_Value__c;
+	}
+	
+	public static Integer getIntegerValue(String sName , integer defaultVal){
+		CW_CampaignListSetting__c campSetting=CW_CampaignListSetting__c.getValues(sName);
+		if(campSetting==null){
+			campSetting= new CW_CampaignListSetting__c();
+			campSetting.Name=sName;
+			campSetting.Integer_Value__c=defaultVal;
+			insert campSetting;
+		}
+		return Integer.ValueOf(campSetting.Integer_Value__c);
+	}
+
+	public static String getTextValue(String sName , String defaultVal){
+		CW_CampaignListSetting__c campSetting=CW_CampaignListSetting__c.getValues(sName);
+		if(campSetting==null){
+			campSetting= new CW_CampaignListSetting__c();
+			campSetting.Name=sName;
+			campSetting.Text_Value__c=defaultVal;
+			insert campSetting;
+		}
+		return campSetting.Text_Value__c;
+	}
+	
+	public static void setDateTimeValue(String sName , DateTime defaultVal){
+		CW_CampaignListSetting__c campSetting=CW_CampaignListSetting__c.getValues(sName);
+		if(campSetting==null){
+			campSetting= new CW_CampaignListSetting__c();
+			campSetting.Name=sName;
+		}
+		campSetting.DateTime_Value__c=defaultVal;
+		upsert campSetting;
+		
+	}
+	
+	public static void setDateValue(String sName , Date defaultVal){
+		CW_CampaignListSetting__c campSetting=CW_CampaignListSetting__c.getValues(sName);
+		if(campSetting==null){
+			campSetting= new CW_CampaignListSetting__c();
+			campSetting.Name=sName;
+		}
+		campSetting.Date_Value__c=defaultVal;
+		upsert campSetting;
+	
+	}
+	
+	public static void setIntegerValue(String sName , integer defaultVal){
+		CW_CampaignListSetting__c campSetting=CW_CampaignListSetting__c.getValues(sName);
+		if(campSetting==null){
+			campSetting= new CW_CampaignListSetting__c();
+			campSetting.Name=sName;
+		}
+		campSetting.Integer_Value__c=defaultVal;
+		upsert campSetting;
+
+	}
+
+	public static void setTextValue(String sName , String defaultVal){
+		CW_CampaignListSetting__c campSetting=CW_CampaignListSetting__c.getValues(sName);
+		if(campSetting==null){
+			campSetting= new CW_CampaignListSetting__c();
+			campSetting.Name=sName;
+		}
+		campSetting.Text_Value__c=defaultVal;
+		upsert campSetting;
+	}
+	
+	public static DateTime DateTimeValue(String sName , DateTime defaultVal){
+		CW_CampaignListSetting__c campSetting=CW_CampaignListSetting__c.getValues(sName);
+		if(campSetting==null){
+			return defaultVal;
+		}
+		return campSetting.DateTime_Value__c;
+	}
+	
+	public static Date DateValue(String sName , Date defaultVal){
```

Comment:
Methods should use the `lowerCamelCase` naming convention. I suggest to rename accordingly.